core/config.cpp

```diff
@@ -62,80 +62,112 @@ void FuzzerConfig::overrideFuzzerConfigsViaFile(const char *path) {
 
     // For the following blocks, retrieve the configurations from the TOML file and override the
     // default configurations if they comply with the constraints.
-    int maxEntryGenCntConfig = tomlConfig["maxEntryGenCnt"].value_or(0);
-    if (maxEntryGenCntConfig <= 0) {
-        error(
-            "P4RuntimeSmith: The maximum number of entries to generate must be provided and must "
-            "be a non-negative integer.");
+    auto maxEntryGenCntNode = tomlConfig["maxEntryGenCnt"];
```

Comment:
Oh I see the problem. Just do not set the value if the object is not defined in the toml. This way you automatically use the default value. 
If the config file is missing some parameter, that is ok. 

But in the test a value you have defined **must** be present in the toml, this is why you should fail there. Does this make sense to you?

Reply:
Ah, I see! I will make the modifications and see if I understand what you've said correctly. After that, I will do a rebase to resolve the PR conflict.

Reply:
Update: I've removed the configuration-overriding-related else clauses and preserve the if clauses in the `overrideFuzzerConfigsViaFile` and `overrideFuzzerConfigsViaString` functions in core/config.cpp since I conjecture that instead of (silently) allowing users to inject configurations that are ill-typed, it may be a better idea to throw an error telling them, for instance, a configuration provided must be an integer.